sdk/core/azure-core-serializer-json-gson/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.0.0-beta.2 (Unreleased)
```

Comment:
@alzimmermsft this was the library that hiccupped when core was released. I'm going to leave this changelog entry in this PR.